crypto-app/src/shared/types.ts

```diff
@@ -0,0 +1,7 @@
+export enum SelectedPage {
+    Home = "home",
+    About = "about",
+    LiveRates = "liverates",
+    ContactUs = "contactus"
+    
+  }
```

Comment:
This is not typing, this is a simple const. create a consts directory and add this there.

```
const SelectedPage = {
  Home: "home",
  About: "about",
  LiveRates: "liverates",
  ContactUs: "contactus"
};
```